tests/www/eligibility_views/test_forms.py

```diff
@@ -48,53 +45,50 @@ def test_valid_for_siae(self):
             f"{criterion3.key}": "true",
             f"{criterion4.key}": "true",
         }
-        form = AdministrativeCriteriaForm(user, siae=company, data=form_data)
+        form = AdministrativeCriteriaForm(False, siae=company, data=form_data)
```

Comment:
```suggestion
        form = AdministrativeCriteriaForm(is_authorized_prescriber=False, siae=company, data=form_data)
```
etc pour les suivants :sweat_smile: